src/SFA.DAS.ApprenticeFeedback/Application/Queries/GetApprentice/GetApprenticeQueryHandler.cs

```diff
@@ -21,7 +21,11 @@ public async Task<GetApprenticeResult> Handle(GetApprenticeQuery request, Cancel
         {
             var result = await _apiClient.Get<GetApprenticeResponse>(new GetApprenticeRequest(request.ApprenticeId));
 
-            if (result == null)
+            var apprenticePreferences =
```

Comment:
You can tweak this to be slightly more efficient, rather than awaiting each call, you can do a Task.WhenAll, and still end up with the same result.